### DISCUSSION TITLE
Screenshot saved as img?

### DISCUSSION THREAD

**Discussion by Ovi-bear** [MarcusWichelmann/MarcusW.VncClient#10] [Q&A] [answered]
how I capture screenshot and save it as image?

**Comment by MarcusWichelmann** [comment, marked as answer]
You could implement a custom render target that takes the received framebuffers and writes each frame as JPEG/PNG to the disk. If you need only one frame, you could close the connection right after that.
Please check out https://github.com/MarcusWichelmann/MarcusW.VncClient#manual-integration and as a reference implementation: https://github.com/MarcusWichelmann/MarcusW.VncClient/blob/master/src/MarcusW.VncClient.Avalonia/RfbRenderTarget.cs

This approach should even work in console applications, no UI framework required.
